cfe_internal/enterprise/federation/federation.cf

```diff
@@ -72,6 +74,9 @@ bundle agent config
         if => strcmp( "feeder", "$(data[remote_hubs][$(remotes)][role])" );
 
   classes:
+    "am_off" expression => or("!enabled", strcmp("off", "$(data[target_state])"))
+      scope => "namespace";
```

Comment:
I assume this is OK considering your other comment and that this is the "pattern" the other am_* classes are using. Rather than below having to say config:am_off.

Reply:
Yeah this is fine. I was just wondering where am_off was used. I saw it later (inside this cfengine_entrpriese_federation namespace) thats mostly what I was wondering about ... is there another bundle using this class? If not, then no need to namespace it (making it global).

Reply:
Do you think we should change this then? Many of the other classes am_* are not used externally either. Only these two are currently used:

```
./cfe_internal/enterprise/CFE_hub_specific.cf:485:    cfengine_enterprise_federation:am_superhub::
./cfe_internal/enterprise/CFE_hub_specific.cf:490:    !cfengine_enterprise_federation:am_superhub::
```